packable/packable-derive-test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "packable-derive-test"
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
Sounds more like a minor to me tbh

Reply:
indeed: https://doc.rust-lang.org/cargo/reference/semver.html#item-new